### PR TITLE
Pass a HOME environment variable to all rabbitmq execute blocks

### PR DESCRIPTION
**Pull request by RoboticCheese**
I ran into https://github.com/jjasghar/rabbitmq/issues/334 today when rolling some new Rabbitmq servers. Since the `ShellOut`s are already ensuring the presence of a `HOME` environment variable, it seemed logical to have the `execute` blocks do the same. Agree/disagree/thoughts?

Thanks!

### CODE REVIEW

**Comment by jjasghar** [comment]
Looks good to me.

Just need another :+1:

File: providers/parameter.rb [hunk 1]
```diff
@@ -51,6 +51,7 @@ def parameter_exists?(vhost, name)
 
     execute "set_parameter #{parameter}" do
       command cmd
+      environment 'HOME' => ENV.fetch('HOME', '/root')
```
Comment (michaelklishin) [inline providers/parameter.rb]: Wouldn't `/var/lib/rabbitmq` be a better fallback value?

Comment (RoboticCheese) [inline providers/parameter.rb]: If so, should I change it to that in the places where `/root` was already being set as well?

Comment (michaelklishin) [inline providers/parameter.rb]: I think so. RabbitMQ or its CLI tools are typically do not run as `root`. Granted, `/var/lib/rabbitmq` isn't portable but it's the most sensible fallback value I can come up with.

**Comment by RoboticCheese** [comment]
At @michaelklishin's suggestion, updated all the fallback home dirs to `/var/lib/rabbitmq`.

**Pull request merged**